src/mono/wasm/test-main.js

```diff
@@ -361,14 +361,19 @@ async function loadDotnet(file) {
             return require(file);
         };
     } else if (is_browser) { // vanila JS in browser
-        loadScript = function (file) {
-            // this is callback we have in CJS version of the runtime
+        loadScript = async function (file) {
```

Comment:
Is there a reason for async when it still returns promise and there isn't any await?